tools/cli/admin_kafka_commands.go

```diff
@@ -469,13 +470,14 @@ func doRereplicate(
 			EndVersion:    endEventVersion,
 		},
 	); err != nil {
-		ErrorAndExit("Failed to resend ndc workflow", err)
+		return ErrorAndPrint("Failed to resend ndc workflow", err)
 	}
 	fmt.Printf("Done rereplication for wid: %v, rid:%v \n", wid, rid)
+	return nil
 }
 
 // AdminRereplicate parses will re-publish replication tasks to topic
-func AdminRereplicate(c *cli.Context) {
+func AdminRereplicate(c *cli.Context) error {
 	sourceCluster := getRequiredOption(c, FlagSourceCluster)
```

Comment:
we have a bunch of these, which are still `ErrorAndExit` internally...
any idea if some/most/all can be switched over to a
```go
&cli.TypeFlag{
  Required: true,
}
```
or if that doesn't work for some reason?  (e.g. I kinda assume can't as it's less flexible, for required combinations)

Reply:
This is more flexible. A flag could be required by one command but not the other. I'll leave it as it is since it might cause unwanted issues